packages/block-editor/src/components/link-control/index.js

```diff
@@ -260,11 +261,27 @@ function LinkControl( {
 				/>
 			) }
 
-			<LinkControlSettingsDrawer
-				value={ value }
-				settings={ settings }
-				onChange={ onChange }
-			/>
+			<div className="block-editor-link-control__tools">
+				<LinkControlSettingsDrawer
+					value={ value }
+					settings={ settings }
+					onChange={ onChange }
+				/>
+				{ onRemove &&
```

Comment:
What do you think instead of allowing a function prop that does a very specific thing, we allow a render props here and let the component consumer put whatever they like here? Something like `renderSecondaryAction`. Might be premature optimization and not worth it though.

Reply:
Im trying to intentionally limit the API here. The Link UI is a major component and I feel it needs tight control over its output. Id prefer to provide a specfic UI here rather than open it up to any UI being added by consumers.